README.md

```diff
@@ -63,6 +63,8 @@ adapter crates to assist in propagating state and instrumenting applications.
 
 In particular, the following crates are likely to be of interest:
 
+- [`opentelemetry-http`] provides an interface for injecting and extracting
```

Comment:
`opentelemetry-http` and `http` don't have corresponding links

```markdown
[`opentelemetry-http`]: https://crates.io/crates/opentelemetry-http
[`http`]: https://crates.io/crates/http
```

Reply:
Still missing the a corresponding link for `http` 🙂 

```markdown
[`http`]: https://crates.io/crates/http
```

Reply:
ok should have 'em all now 😅

Reply:
Looks good 😄